system/include/webgl/webgl1_ext.h

```diff
@@ -228,6 +228,17 @@ WEBGL_APICALL void GL_APIENTRY emscripten_webgl_getSupportedAstcProfiles(GLsizei
 #define GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT3_EXT 0x8C4E
 #define GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT5_EXT 0x8C4F
 
+// 40. https://www.khronos.org/registry/webgl/extensions/WEBGL_multi_draw/
+// https://chromium.googlesource.com/angle/angle/+/master/extensions/ANGLE_multi_draw.txt
+WEBGL_APICALL void GL_APIENTRY emscripten_glMultiDrawArraysANGLE(GLenum mode, const GLint* firsts, const GLsizei* counts, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY emscripten_glMultiDrawArraysInstancedANGLE(GLenum mode, const GLint* firsts, const GLsizei* counts, const GLsizei* instanceCounts, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY emscripten_glMultiDrawElementsANGLE(GLenum mode, const GLsizei* counts, GLenum type, const GLsizei* offsets, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY emscripten_glMultiDrawElementsInstancedANGLE(GLenum mode, const GLsizei* counts, GLenum type, const GLsizei* offsets, const GLsizei* instanceCounts, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY glMultiDrawArraysANGLE(GLenum mode, const GLint* firsts, const GLsizei* counts, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY glMultiDrawArraysInstancedANGLE(GLenum mode, const GLint* firsts, const GLsizei* counts, const GLsizei* instanceCounts, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY glMultiDrawElementsANGLE(GLenum mode, const GLsizei* counts, GLenum type, const GLsizei* offsets, GLsizei drawcount);
+WEBGL_APICALL void GL_APIENTRY glMultiDrawElementsInstancedANGLE(GLenum mode, const GLsizei* counts, GLenum type, const GLsizei* offsets, const GLsizei* instanceCounts, GLsizei drawcount);
```

Comment:
Here also let's name these with `WEBGL` suffix. I believe the `ANGLE` function signatures should be placed in `GLES2/gl2ext.h` header, as they are GLES extensions. I.e. people who compile GLES2/GLES3 codebases and target this extension without targeting WebGL headers would not be including `webgl1_ext.h` header file.